src/import/trace-event.ts

```diff
@@ -100,15 +100,15 @@ function selectQueueToTakeFromNext(
   if (bts < ets) return 'B'
   if (ets < bts) return 'E'
 
-  // If we got here, the 'B' event queue and the 'E' event queue have events at
+   // If we got here, the 'B' event queue and the 'E' event queue have events at
```

Comment:
Super tiny nit: can you revert this whitespace change? it looks like it was accidental

Reply:
Yep - also I ran `prettier` and a bunch of files changed so I didn't commit that, I can push up a follow up commit